packages/graphql-server/src/resolvers/contact/api.ts

```diff
@@ -54,8 +54,55 @@ export const callGetContactsAPI = async (args: GetContactsArgs, context: ServerC
   }
 }
 
-// TODO: will call create contact API
-export const callCreateContactAPI = async (contact: CreateContactArgs) => {
-  logger.info('createContact', { contact })
-  return {}
+export const callCreateContactAPI = async (args: CreateContactArgs, context: ServerContext) => {
```

Comment:
Can you add a return type defintion

Reply:
Hi @willmcvay ,
In GraphQL server, we've already had `schema.graphql` file where we will define params and return type.
In `api.ts` file, we just call the fetcher and return the response or the errors if something's wrong. Does it still necessary to define return types for functions in `api.ts`

Reply:
This is a public endpoint, I want us to be as tight and explicit about what the code is doing as possible. In this case I can't see from looking at the code what type createResponse is at line 73 - if I can't tell this, my users can't either